defactor-com/ui-kit
# Feat/update the lendingborrowing component 85

## Update the lendingborrowing component

### What does this PR do?

- Resolve #119
- Resolve #129 

### Steps to test

1. Run project
2. Check changes

Before:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/e22a0292-4f4a-43e6-9b83-781f571f4110)

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/45446324-2091-4637-b19f-f34df57bb194)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/d7045443-6ef0-4860-a2cc-b07899eab4c1)

#### CheckList
